opossum-tool/OpossumUI
# feat: show origin source for manual attributions

### Summary of changes

Add chips showing the origin source of of manual attributions in the attribution column
Add tooltips for chips in the attribution column

### Context and reason for change

See #2327 

### How can the changes be tested

- run 'yarn start'
- open 'opossum_input.json' or  'opossum_input.opossum'
- navigate to '/index.tsx'
- click through the different attributions
- check that each attribution has an 'origin source' chip
- check that the origin sources match with the sources of the corresponding signals

![origin_source_chip](https://github.com/opossum-tool/OpossumUI/assets/89708272/bc34432c-033d-4362-bbf2-b328e236bb11)
